pkgs/applications/office/jameica/default.nix

```diff
@@ -35,13 +35,17 @@ stdenv.mkDerivation rec {
     sha256 = "1x9sybknzsfxp9z0pvw9dx80732ynyap57y03p7xwwjbcrnjla57";
   };
 
+  dontWrapGApps = true;
+
   # there is also a build.gradle, but it only seems to be used to vendor 3rd party libraries
   # and is not able to build the application itself
   buildPhase = ''
     (cd build; ant -Dsystem.version=${version} init compile jar)
   '';
 
   installPhase = ''
+    gappsWrapperArgsHook
```

Comment:
That is strange. Normally we don't need to run this. Are we missing some runHook's? If so please add them instead.

Reply:
ill check that. 
i did already check if it does make a difference, as the `wrapGAppsHook` is already added.
And yes it makes a difference, when `gappsWrapperArgsHook` is called, it additionally adds the needed `GDK_PIXBUF_MODULE_FILE` and `XDG_DATA_DIRS` which are the variables i am looking for. But atm idk why they are missing when just the hook does its job.

Reply:
okay, the `wrapGAppsHook` fills the `gappsWrapperArgsHook` in preFixup, which is the reason, that the args are unset when using them in `installPhase`. 
So i moved the `makeWrapper` call to a phase after that (its mostly done in fixup or postFixup), in my case i opted for fixupPhase.

Hopefully thats okay, if its not, please let me know.